gallery/src/main.scss

```diff
@@ -77,6 +77,24 @@
   }
 }
 
+.gallery-example-live {
+  $checker-size: 12px;
+  $checker-color: #e9e9e9;
+  $checker-bg-color: #fff;
+  $checker-layer-1-positioning: top left/$checker-size $checker-size;
+  $checker-layer-2-positioning: #{$checker-size / 2} #{$checker-size / 2}/$checker-size $checker-size;
+  $checker-layer-3-positioning: -#{$checker-size / 2} -#{$checker-size / 2}/$checker-size $checker-size;
+
+  padding: $nx-spacing-l;
+
+  // create a checkerbox pattern for the background so that the edges and background of the example are clear
+  background: repeat $checker-layer-3-positioning linear-gradient(45deg, transparent 75%, $checker-color 75%),
+              repeat $checker-layer-2-positioning linear-gradient(45deg, $checker-color 25%, transparent 25%),
+              repeat $checker-layer-1-positioning
+                  linear-gradient(45deg, $checker-color 25%, transparent 25% 75%, $checker-color 75%),
+              $checker-bg-color;
```

Comment:
The way that this works is admittedly non-obvious; I largely did it like this as a learning experience.  I'd be willing to replace it with a png or svg background if desired.

Reply:
> The way that this works is admittedly non-obvious; I largely did it like this as a learning experience. I'd be willing to replace it with a png or svg background if desired.

It's complicated but all of the variables presented allow so much latitude for customization that I think it's worth it.